clang/tools/clang-scan-deps/ClangScanDeps.cpp

```diff
@@ -288,9 +288,7 @@ handleTreeDependencyToolResult(llvm::cas::CASDB &CAS, const std::string &Input,
     return true;
   }
   OS.applyLocked([&](llvm::raw_ostream &OS) {
-    OS << "tree ";
-    llvm::cantFail(CAS.printCASID(OS, *MaybeTree));
-    OS << " for '" << Input << "'\n";
+    OS << "tree " << *MaybeTree << " for '" << Input << "'\n";
```

Comment:
How does this work? TreeRef doesn't have << operator. This breaks auto merger.

Reply:
Pushed in 007f2611a4ae702885e47eff4f5ddcef4748901c. Also fixes `llc` build which had (a) a related issue and (b) an unclosed parenthesis?

Does the Linux build not actually do anything?